mm2src/coins/lp_coins.rs

```diff
@@ -2091,6 +2091,10 @@ pub enum WithdrawFee {
         gas_limit: u64,
         gas_price: f64,
     },
+    CosmosGasPriority {
+        gas_limit: u64,
+        gas_price_option: TendermintPriorityFeeOption,
+    },
```

Comment:
Can't we extend existing option `CosmosGas`?

Reply:
Actually I did this like EthGasEip1559 option (which is also a dedicated variant to set priority tx fee for eth).
Also, if we combine mutually exclusive `gas_price: f64` and `gas_price_option: TendermintPriorityFeeOption` in CosmosGas, this would require extra checks in the code.